config/samples/gcp/gcp_openshift_route.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: externaldns.olm.openshift.io/v1alpha1
+kind: ExternalDNS
+metadata:
+  name: sample-gcp
+spec:
+  domains:
+  - filterType: Include
+    matchType: Exact
+    name: misalunkgcp.qe.gcp.devcluster.openshift.com
+  provider:
+    type: GCP
+  source:
+    hostnameAnnotation: Allow
```

Comment:
Hostname annotation is not really necessary for OpenShift routes and it's not the most secure option so it's better to know show it as a recommended sample.